cmd/kubelet/app/server.go

```diff
@@ -1156,12 +1156,12 @@ func RunDockershim(f *options.KubeletFlags, c *kubeletconfiginternal.KubeletConf
 
 	// Initialize network plugin settings.
 	pluginSettings := dockershim.NetworkPluginSettings{
-		HairpinMode:        kubeletconfiginternal.HairpinMode(c.HairpinMode),
-		NonMasqueradeCIDR:  f.NonMasqueradeCIDR,
-		PluginName:         r.NetworkPluginName,
-		PluginConfDir:      r.CNIConfDir,
-		PluginBinDirString: r.CNIBinDir,
-		MTU:                int(r.NetworkPluginMTU),
+		HairpinMode:         kubeletconfiginternal.HairpinMode(c.HairpinMode),
+		NonMasqueradeCIDR:   f.NonMasqueradeCIDR,
+		PluginName:          r.NetworkPluginName,
+		PluginConfDirString: r.CNIConfDir,
```

Comment:
can cni.SplitDirs() be used directly here over the PluginConfDirs field instead?